src/types/repoInfo.ts

```diff
@@ -15,10 +15,10 @@ export type GitHubRepositoryData = {
 }
 
 export type RepositoryData = {
-  lastUpdated: GitHubRepositoryData["pushed_at"]
-  permissions: GitHubRepositoryData["permissions"]
+  lastUpdated?: GitHubRepositoryData["pushed_at"]
+  permissions?: GitHubRepositoryData["permissions"]
   repoName: GitHubRepositoryData["name"]
-  isPrivate: GitHubRepositoryData["private"]
+  isPrivate?: GitHubRepositoryData["private"]
```

Comment:
hmm actually removing isPrivate has implications specifically for loading homepages on private repos - without this param available the images on editHomepage don't load properly. I feel like this can be a future fix though since the affected repos should be small, but let's check with prod ops first if any existing email login sites are private

Reply:
We should then be tracking this info in our DB right?